4-appfactory/modules/app-group-baseline/main.tf

```diff
@@ -80,7 +80,7 @@ module "app_env_project" {
   random_project_id        = true
   random_project_id_length = 4
   billing_account          = each.value.billing_account
-  name                     = substr("eab-${var.application_name}-${each.key}", 0, 25) # max length 30 chars
+  name                     = substr("eab-${var.application_acronym}-${var.service_name}-${each.key}", 0, 25) # max length 30 chars
```

Comment:
we currently have an issue in naming on env projects:

eab-transactionhistory-pr-XXXX
eab-balancereader-product-XXXX

using the app_acronym here would yield something like

eab-cb-transactionhistory-XXXX
eab-cb-balancereader-produ-XXXX


I suggest we use something like this for the name

`substr("${substr(each.key, 0, 1)}-${var.application_acronym}-${var.service_name}", 0, 25)`

This would yield something like `p-cb-transactionhistory-XXXX`

what do you guys think?

Reply:
other idea that can help solve this issue:

for project id's lets also use service acronyms naming convention, for example instead of using `transactionhistory` -> `t-hst`, 'userservice' -> 'u-svc' or something like this